cfg/rippled-example.cfg

```diff
@@ -283,12 +283,14 @@
 #       ssl_cert
 #
 #           Specifies the path to the SSL certificate file in PEM format.
-#           This is not needed if the chain includes it.
+#           This is not needed if the chain includes it.  Use ssl_chain if
+#           your certificate includes one ore more intermediates
 #
 #       ssl_chain
 #
 #           If you need a certificate chain, specify the path to the
 #           certificate chain here. The chain may include the end certificate.
+#           This must be used if the certificate includes intermediates
```

Comment:
```suggestion
#           This must be used if the certificate includes intermediates.
```
Most sentences in this file are ended with a period (`.`) so I figure we should remain consistent.